code-challenges.js

```diff
@@ -44,11 +85,29 @@ var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+const vowelOnly = (string) => {
+  let arrayVowel = string.split("")
+  return arrayVowel.filter(value => value !== "a" && value !== "A" && value !== "e" && value !== "E" && value !== "i" && value !== "I" && value !== "o" && value !== "O" && value !== "u" && value !== "U").join("")
```

Comment:
Dude! Nice work with this! You thought about the edge cases and considered the capital vowels! Yes! 

Another solution to this, to keep your code a little shorter, you could use `.toLowerCase()` on each value and not have to worry about the uppercase letters. Just something to think about- but this code is bangin'!